src/xinterpreter.cpp

```diff
@@ -357,7 +362,32 @@ __get_cxx_version ()
 
     void interpreter::init_includes()
     {
+        // Add the standard include path
         Cpp::AddIncludePath((xeus::prefix_path() + "/include/").c_str());
+
+        // Get include paths from environment variable and use empty string if not set
+        const char* non_standard_paths = std::getenv("XEUS_SEARCH_PATH");
+        if (!non_standard_paths)
+        {
+            non_standard_paths = "";
+        }
+
+#ifdef _WIN32
+        const char path_separator = ';';
+#else
+        const char path_separator = ':';
+#endif
+
+        // Split and add each non-empty path
+        std::istringstream stream(non_standard_paths);
+        std::string path;
+        while (std::getline(stream, path, path_separator))
+        {
+            if (!path.empty())
+            {
+                Cpp::AddIncludePath(path.c_str());
+            }
+        }
```

Comment:
```suggestion
        while (std::getline(stream, path, path_separator))
            if (!path.empty())
                Cpp::AddIncludePath(path.c_str());
```

Reply:
Apologies, I initially thought the braces were necessary for the clang-tidy format check to pass. I've now edited and correctly formatted the file.